controllers/appwrapper_controller.go

```diff
@@ -90,9 +89,17 @@ var kubeClient *kubernetes.Clientset
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.11.0/pkg/reconcile
 func (r *AppWrapperReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+
+	if ocmClusterID == "" && useMachineSets == false {
+		err := getAccessTokens(r)
+		if err != nil {
+			useMachineSets = true
```

Comment:
we should probably log the error. I think users will use either machineset or machinepool.